cutil/parser/fsfileresource/localfs_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fsfileresource
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestLocalResource_Read(t *testing.T) {
```

Comment:
I think there should also be some tests to see if throwing errors works a intended. So tests of resources that don't exists. ATM it looks like you only test with existing files if i'm not mistaking. This goes the same for the http resources tests.